jbehave-support-core/src/main/java/org/jbehavesupport/core/web/WebScreenshotTypes.java

```diff
@@ -0,0 +1,24 @@
+package org.jbehavesupport.core.web;
+
+import org.springframework.stereotype.Component;
+
+@Component
+public class WebScreenshotTypes {
```

Comment:
I would probably do just a WebScreenshotType enum directly without having enum in class (plus the component is not needed since we dont autowire it anywhere)

Reply:
Class is gone, I renamed enum to WebScreenshotType.